guides/common/modules/ref_ports-and-firewall-requirements.adoc

```diff
@@ -97,8 +97,8 @@ Remote Execution result upload
 ifdef::satellite[]
 ifeval::["{mode}" == "connected"]
 | 443 | TCP | HTTPS | console.redhat.com | Red{nbsp}Hat Cloud plugin API calls |
-| 443 | TCP | HTTPS | Red{nbsp}Hat Portal | SOS report | Assisting support cases (optional)
-| 443 | TCP | HTTPS | Red{nbsp}Hat CDN | Content Sync | Red{nbsp}Hat CDN
+| 443 | TCP | HTTPS | link:https://access.redhat.com/solutions/1179133[Red{nbsp}Hat Portal] | SOS report | Assisting support cases (optional)
+| 443 | TCP | HTTPS | link:https://access.redhat.com/articles/1525183[Red{nbsp}Hat CDN] | Content Sync | Red{nbsp}Hat CDN
```

Comment:
```suggestion
| 443 | TCP | HTTPS | cdn.redhat.com | Content Sync | https://access.redhat.com/articles/1525183[Red{nbsp}Hat CDN]
| 443 | TCP | HTTPS | api.access.redhat.com | SOS report | Assisting support cases filed through the https://access.redhat.com/solutions/1179133[Red{nbsp}Hat Customer Portal] (optional)
```

1. Mention FQDN of the service instead of service name for better clarity. (I was recommended to do this by a SME.)
2. Links work better in the description column.
3. `link:` syntax not required, because URLs are recognized automatically.
4. Switch the order so that `api.access` and the following `cert-api.access` are next to each other in the table.